app/api/users/users.js

```diff
@@ -178,7 +177,7 @@ export default {
     }
     const _user = await model.save({
       ...user,
-      password: await encryptPassword(random()),
+      password: await encryptPassword(user.password),
```

Comment:
Good catch @grafitto, please could you add a test to cover this, if I reverted the changes all tests are passing.

Reply:
I believe the tests are already testing the password as opposed to `random()`

Reply:
Actually no, I saw some tests that check the password, but since all of them are passing even without the new changes, there is no one that covers this case.